CHANGELOG.md

```diff
@@ -0,0 +1,48 @@
+# Changelog
+
+## v0.2.0
+
+This is a hardfork release for the opBNB Testnet called Fermat.
+It will be activated at block height 13063000, expected to occur on November 14, 2023, at 6 AM UTC.
+
+### User Facing Changes
+
+- The L1 gas price of all L2 transactions will be fixed(3 Gwei by default and 5 Gwei for Testnet).
+- Introduce a new type of RPC kind called `bsc_fullnode`. To enable it, include the parameter `--l1.rpckind=bsc_fullnode` if the layer 1 endpoint supports the `eth_getTransactionReceiptsByBlockNumber` API. This will significantly enhance the performance of retrieving L1 receipts (#63).
+- The rollup configuration for opBNB Mainnet and Testnet has been added to the code. You can now use the `--network=opBNBTestnet` or `--network=opBNBMainnet` flag instead of `--rollup.config=./rollup.json` to specify the rollup configuration for the op-node. (#65)
```

Comment:
If a node still runs with `--rollup.config=./rollup.json` and the `./rollup.json` doesn't update to date, will the node fork against the main network?

Reply:
It will retrieve the block chain id in `rollup.json` and overwrite the config it the chain id matches Mainnet or Testnet